search/tests/test_sorting.py

```diff
@@ -0,0 +1,117 @@
+"""
```

Comment:
I don't see why constants have to be tested at all, when it doesn't do anything for test coverage.